src/components/Todos.jsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from "react"
+import { AiOutlinePlus, AiFillDelete, AiOutlineCheck, AiOutlineClose } from "react-icons/ai"
+
+export default function Todos() {
+    const [text, setText] = useState('')
+    const [displayText, setDisplayText] = useState([])
+
+    const handleChange = (event) => {
+        setText(event.target.value)
+    }
+
+    const handleClick = () => {
+        if (!text.trim()) return
+        text ? setDisplayText([...displayText, { value: text, isChecked: false }]) : null
+        setText('')
+    }
+
+    const handleCheckboxChange = (index) => {
+        setDisplayText((prevDisplayText) => {
+            const newDisplayText = [...prevDisplayText];
+            newDisplayText[index] = {
+                ...newDisplayText[index],
+                isChecked: !newDisplayText[index].isChecked,
+            }
```

Comment:
You can use destructuring to simplify your code and make it more readable. For example, instead of prevDisplayText, you can destructure the displayText state directly: 
```suggestion
            const newDisplayText = [...displayText];
    newDisplayText[index] = {
      ...displayText[index],
      isChecked: !displayText[index].isChecked,
    };
```